### PR TITLE
Change contribution guidance

### PR DESCRIPTION
**Pull request by owenm6**
Give additional details and update the markup.

### CODE REVIEW

**Comment by jbuller** [comment]
'please remove the Home Office name and link back to this repo giving appropriate credit.' seems an odd phrasing.
Should it be a must not a please? Not citing us means not giving credit.  Will we have a template that has a softer 'based on designs by Home Office' or similar credit? Or specifically not providing such a template to deter further derrivations.

**Comment by SueSpevack** [comment]
We agreed that external organisations can use our template, but not associate it with using the Home Office logo - hence why Micheal has included this piece of guidance. We won't be producing any more posters based on this template - this will only be for external organisations to use.

File: README.md [hunk 1]
```diff
@@ -1,45 +1,38 @@
...
 # Home Office Posters
-Home Office repository of posters covering different topics - research, access needs, accessibility, design.
+Home Office repository of posters covering different topics - research, access needs, accessibility and design.  
 
+## Contributions and contact
 
+If you want to contribute a translation of one of our posters, we would really appreciate your help.
 
-**See ALL the posters**
+You are free to adapt the posters under [the licence](https://creativecommons.org/licenses/by-nc-sa/4.0/). But please remove the Home Office name and link back to this repo giving appropriate credit.
```
Comment (christaylor-hod) [inline README.md]: The last section could be a little clearer (I was initially confused by it saying remove name but add link) - could we seperate out the sentence into bullets, for eg:

You are free to adapt the posters under the following conditions:
- the posters are available under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International licence](https://creativecommons.org/licenses/by-nc-sa/4.0/)
- remove the Home Office name and logo
- give appropriate credit and link back to this repo

Comment (SueSpevack) [reply README.md]: Thanks Chris - this looks great.

**Comment by owenm6** [comment]
Thanks for the helpful comments. If there are no further comments I think it is good to publish these changes @karypun

File: README.md (hunk 1 above)
Comment (owenm6) [reply README.md]: Have adjusted so its clearer like suggested

**Pull request merged**